apps/02-creacion-de-listas/src/create.erl

```diff
@@ -8,13 +8,30 @@
 %% @spec create(Value :: pos_integer()) -> [pos_integer()]
 %% @end
 %%--------------------------------------------------------------------
-create(_Value) ->
-    erlang:throw(not_implemented).
+create(N) when is_number(N)->
+    create(N,[]);
+create(_) ->
+    erlang:throw("NAN").
+
+create(0, List) ->
```

Comment:
All private functions should be together at the end of the file, and arranged alphabetically.